### PR TITLE
Landing nav & styling updates

### PR DESCRIPTION
**Pull request by epwinchell**
Jira: https://issues.redhat.com/browse/RHCLOUD-18708

### CODE REVIEW

File: src/js/App/Sidenav/Navigation/Navigation.scss [hunk 1]
```diff
@@ -1,25 +1,26 @@
 .chr-c-app-title {
   color: white;
-  padding: var(--pf-global--spacer--md) var(--pf-global--spacer--md) var(--pf-global--spacer--md) var(--pf-global--spacer--lg);
+  padding: var(--pf-global--spacer--md) var(--pf-global--spacer--sm) var(--pf-global--spacer--md) var(--pf-global--spacer--lg);
   width: 100%;
   border-bottom: var(--pf-global--spacer--xs) solid var(--pf-global--Color--300);
-  text-transform: capitalize;
+  // text-transform: capitalize;
```
Comment (Hyperkid123) [inline src/js/App/Sidenav/Navigation/Navigation.scss:6]: Do we require this omment?

File: src/js/App/Sidenav/Navigation/Navigation.scss [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
   font-weight: var(--pf-global--FontWeight--semi-bold);
   white-space: normal;
   text-align: left;
-  font-size: var(--pf-global--FontSize--lg);
+  font-size: 17px;
```
Comment (Hyperkid123) [inline src/js/App/Sidenav/Navigation/Navigation.scss:10]: Is it possible to use some of the PF variables instead?

Comment (epwinchell) [reply src/js/App/Sidenav/Navigation/Navigation.scss:10]: @Hyperkid123 No - this is between PF variable sizes.

File: src/js/App/Sidenav/Navigation/Navigation.scss (hunk 1 above)
Comment (epwinchell) [reply src/js/App/Sidenav/Navigation/Navigation.scss]: no - removed

File: src/js/App/Sidenav/Navigation/Navigation.scss (hunk 2 above)
Comment (Hyperkid123) [reply src/js/App/Sidenav/Navigation/Navigation.scss:9]: Do we have to be between the PF sizes?

Comment (epwinchell) [reply src/js/App/Sidenav/Navigation/Navigation.scss:9]: @Hyperkid123 - yes - we can't go bigger and Kendra doesn't want to go smaller (we tested various sizes together)

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**